data/src/main/java/com/teamagam/gimelgimel/data/config/Constants.java

```diff
@@ -3,7 +3,7 @@
 import android.graphics.Bitmap;
 
 public class Constants {
-    public static final String MESSAGING_SERVER_URL = "http://ggmessaging-dev1.herokuapp.com";
+    public static final String MESSAGING_SERVER_URL = "http://ggmessaging-dev2.herokuapp.com";
```

Comment:
you can make changes to files and move those filese to other "change list" in android studio so it would not appear in our version control.